openvmm/hvlite_entry/src/cli_args.rs

```diff
@@ -766,14 +767,15 @@ impl FromStr for FloppyDiskCli {
     }
 }
 
-// (console | stderr | listen=\<path\> | none)
+/// (console | stderr | listen=<path> | none)
```

Comment:
this comment should prob also get tweaked

Reply:
```suggestion
/// (console | stderr | listen=\<path\> | listen=tcp:\<ip\>:\<port\> | none)
```